### ISSUE TITLE
Title property is missing in result PDF

### ISSUE THREAD

**Issue by mgm-rwagner** [danfickle/openhtmltopdf#707]
I tried the example from the wiki page (https://github.com/danfickle/openhtmltopdf/wiki/PDF-Accessibility-(PDF-UA,-WCAG,-Section-508)-Support) with the 1.0.8 release and the accessibility checker returns that the title is missing.

![grafik](https://user-images.githubusercontent.com/75724274/119643544-268bf400-be1c-11eb-8bee-080ddc6f4d8e.png)

Maybe i do something wrong but with the 1.0.3 release it was working.

Is there something like a workaround? I need it in an project very soon.

**Comment by syjer** [comment]
Hi @mgm-rwagner , looks like the title XMP metadata is not set according to the checker:

![detail](https://user-images.githubusercontent.com/498146/119667073-3dd7db00-be36-11eb-98e5-90d43e44d936.PNG)

I'll have a deeper look.

**Comment by syjer** [comment]
hi @mgm-rwagner , I did a small test, you need to set the "PdfAConformance".

In you builder:

`builder.usePdfAConformance(PdfRendererBuilder.PdfAConformance.PDFA_3_U);`

You get:

![ok](https://user-images.githubusercontent.com/498146/119668608-9e1b4c80-be37-11eb-8d38-3ae98e02d2c0.PNG)

I'll update the documentation, as it was missing this line of code.

**Comment by syjer** [comment]
About the reason: you can see in https://github.com/danfickle/openhtmltopdf/blob/open-dev-v1/openhtmltopdf-pdfbox/src/main/java/com/openhtmltopdf/pdfboxout/PdfBoxRenderer.java#L728

Without a PDF/A set, it will avoid the whole section where it set some part of the xmp metadata.

**Comment by mgm-rwagner** [comment]
Thank you for the fast response. Now it works.

**Comment by danfickle** [comment]
One should be able to create PDF/UA docs that are not also PDF/A compliant, I think. I'll have to do some research on what metadata is required in each of the following circumstances:
+ PDF/UA
+ PDF/A
+ Both
This also shows the perils of not having automated testing as it was correct (I believe) for PDF/UA but not PDF/A before fix was applied for the latter.

BTW. Thanks @syjer for investigating.
